### PR TITLE
Add selective disclosure query to for drivers license credential

### PR DESCRIPTION
**Pull request by jameseaster**
#### _Resolves #58_

---

### What kind of change does this PR introduce?

- Configuration update.

<br/>

### What is the current behavior?

- No current example of selective disclosure with a driver's license credential.

<br/>

### What is the new behavior?

- Include selective disclosure example for driver's license that requests subject's name & DL#.

<br/>

### Does this PR introduce a breaking change?

- No

<br/>

### How has this been tested?

- Locally tested by issuing and verifying DL credential.

<br/>

### Screenshots: n/a

### CODE REVIEW

**Review by dlongley — APPROVED** [review]

File: credentials/utopia-dl/queries.json [hunk 1]
```diff
@@ -20,5 +20,34 @@
...
+  "Driver's License (Name & Number Only)": {
+    "type": "QueryByExample",
+    "credentialQuery": {
+      "reason": "Please present your ISO 18013 Drivers License Verifiable Credential(s) to complete the verification process.",
+      "example": {
+        "@context": [
+          "https://www.w3.org/2018/credentials/v1",
+          "https://www.w3.org/ns/credentials/examples/v2",
+          "https://w3id.org/vdl/v1",
+          "https://w3id.org/vdl/aamva/v1"
```
Comment (dlongley) [inline credentials/utopia-dl/queries.json:33]: I wonder if the aamva one is needed for this query or not. This is something we'll want to clean up over time -- ensuring we only request the needed contexts and that the wallet only delivers those requested.

Comment (msporny) [reply credentials/utopia-dl/queries.json:33]: I don't think we need any AAMVA defined terms in the example.

Comment (jameseaster) [reply credentials/utopia-dl/queries.json]: Updated: aea7f8c

File: credentials/utopia-dl/queries.json [hunk 2]
```diff
@@ -20,5 +20,33 @@
...
     }
+  },
+  "Driver's License (Name & Number Only)": {
+    "type": "QueryByExample",
+    "credentialQuery": {
+      "reason": "Please present your ISO 18013 Drivers License Verifiable Credential(s) to complete the verification process.",
+      "example": {
+        "@context": [
+          "https://www.w3.org/2018/credentials/v1",
+          "https://www.w3.org/ns/credentials/examples/v2",
```
Comment (jameseaster) [inline credentials/utopia-dl/queries.json:31]:
```suggestion
```

**Pull request merged**